packages/graphql/package.json

```diff
@@ -16,7 +16,8 @@
   "browser": "dom.js",
   "files": [
     "node.js",
-    "dom.js"
+    "dom.js",
```

Comment:
Also: The `commands/` folder is missing in `"files"`?

Reply:
I should probably simply got to bed... 😴